### PR TITLE
Resnet paper with a new *classification* category

### PR DESCRIPTION
**Pull request by jodoin**

### CODE REVIEW

File: deep-learning/_posts/2017-03-01-alexnet.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: review
 title: (AlexNet) ImageNet Classification with Deep Convolutional Neural Networks
```
Comment (lemairecarl) [inline deep-learning/_posts/2017-03-01-alexnet.md]: I think that "AlexNet : ImageNet Classification..." would be more readable than using parentheses. If you use a colon you must surround the title with quotes. (not a big deal)

File: deep-learning/_posts/2017-03-16-resnet.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+layout: review
+title: (ResNet) Deep Residual Learning for Image Recognition
```
Comment (lemairecarl) [inline deep-learning/_posts/2017-03-16-resnet.md]: I think that "ResNet : Deep Residual Learning..." would be more readable than using parentheses. If you use a colon you must surround the title with quotes. (not a big deal)

File: deep-learning/_posts/2017-03-16-resnet.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    venue:   "Proceedings of CVPR 2016, p.770-778"
+pdf:   "http://papers.nips.cc/paper/4824-imagenet-classification-with-deep-convolutional-neural-networks.pdf"
+---
+
+ 
+Winner in several tracks of the ILSVRC & COCO 2015 competitions: ImageNet detection, ImageNet localization, COCO detection, and COCO segmentation.  The main novelty of this paper is the use of residual blocks which is a series of 2 or 3 conv layers with a skip connection.
+ 
+![](/deep-learning/images/resnet/resnet2.jpg)
+
+The skip connection works as identity shortcuts to remove the unuseful blocks and thus to build much deeper architectures
```
Comment (lemairecarl) [inline deep-learning/_posts/2017-03-16-resnet.md]: I suggest those changes:

"The skip connection works as an identity shortcut to skip a block when it is not useful for a particular class. This allows to build much deeper architectures."

When a block is skipped, it's not completely useless. If it was the case, we would just remove it.

File: deep-learning/_posts/2017-03-16-resnet.md (hunk 2 above)
Comment (jodoin) [inline deep-learning/_posts/2017-03-16-resnet.md]: done

File: deep-learning/_posts/2017-03-01-alexnet.md (hunk 1 above)
Comment (jodoin) [inline deep-learning/_posts/2017-03-01-alexnet.md]: done

File: deep-learning/_posts/2017-03-16-resnet.md (hunk 3 above)
Comment (jodoin) [inline deep-learning/_posts/2017-03-16-resnet.md]: done

**Pull request merged**